src/item.h

```diff
@@ -327,13 +327,21 @@ class item : public visitable<item>
     /**
      * Returns the default color of the item (e.g. @ref itype::color).
      */
+
     nc_color color() const;
     /**
      * Returns the color of the item depending on usefulness for the player character,
      * e.g. differently if it its an unread book or a spoiling food item etc.
      * This should only be used for displaying data, it should not affect game play.
      */
+
     nc_color color_in_inventory() const;
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    std::string sname( unsigned int quantity = 1 ) const;
+
+    std::string tname_generate_tagtext() const;
+    std::string tname_generate_maintext( unsigned int quantity = 1 ) const;
```

Comment:
These two functions should probably be private as I see no need for code outside of the naming functions to call it. Also the default value for the parameter is never used, all calls to the function supply an explicit value. It's not needed.